tests/rafter/README.md

```diff
@@ -0,0 +1,65 @@
+# Rafter Integration Tests
+
+## Overview
+
+The project is a test scenario for all Rafter subcomponents, such as controllers and the Asset Upload Service.
+
+## Prerequisites
+
+Use the following tools to set up the project:
+
+- [Go](https://golang.org)
+- [Docker](https://www.docker.com/)
+
+## Usage
+
+### Run a local version
+
```

Comment:
Could you add some more context to this part? I'm not sure if it is all clear what should be done, what do you think?

Reply:
Done!